pkg/util/azureclient/azuresdk/azsecrets/client.go

```diff
@@ -0,0 +1,39 @@
+package azsecrets
```

Comment:
Same as above:-
It seems the functions for the client are not used till now anywhere, but would it be good to have a mocks generated for it now?

Reply:
I'll plan to include the mocks for this client as well in my follow-up PR. I didn't end up needing them in unit tests for this PR because of the limited extent to which I used this client.